config_example.ini

```
@@ -16,3 +16,10 @@ twitter_access_token_secret = "ACCESS_TOKEN_SECRET"
 [slack]
 slack_invite_url = "https://join.slack.com/t/nfcore/shared_invite/INVITE_KEY"
 slack_access_token = "ACCESS_TOKEN_SECRET"
+
+; MySQL database details
+host = 'db'
```

I'm not sure that the ini sections have any functional effect (?) but for consistency..

```suggestion
; MySQL database details
[sqldb]
host = 'db'
```